### DISCUSSION TITLE
GitHub Student Pack Application Status Query: Already Submitted 2 Requests - Need Guidance

### DISCUSSION THREAD

**Discussion by pathan985** [community/community#106918] [GitHub Education]
### Hi everyone,

I've recently applied for the **GitHub Student Pack** and encountered a message that says, **"You've already submitted 2 requests."** One of these requests has been **REJECTED**, and the other one is **APPROVED**. 

Additionally, the message states, "**You were last verified as a null on Feb 7, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.**"

To provide a clearer context, I have attached images of the messages received. I'm a bit confused about the status of my application – is it rejected or approved? If anyone has faced a similar situation or has insights on what steps to take next, I'd greatly appreciate your guidance.

![image](https://github.com/community/community/assets/159281432/4cddf98a-5e8a-45f4-80da-34acbfaa2bd0)

![image](https://github.com/community/community/assets/159281432/54d31133-cfbc-49ec-9463-368cb7a92fa5)

**Comment by Lelin07** [comment]
I'm facing the exact same errors even after using university approved active mail id. Do you have any updates on this ?
